docs/Fachschaftsordnung.tex

```diff
@@ -149,6 +149,7 @@ \section{Arbeit des Fachschaftsrats}
 \begin{itemize}
 	\item Der FSR gewährleistet eine angemessene Beratung der Studierenden und sorgt für Präsenzzeiten. Die Präsenzzeiten werden in einem Plan festgehalten, der am Anfang eines jeden Semesters aufzustellen ist. In der Vorlesungszeit ist ein Minimum von zwei Tagen in der Woche mit insgesamt mindestens vier Stunden Präsenzzeit zu gewährleisten. Die Mitglieder des FSR sorgen für den Präsenzdienst. Während des Präsenzdienstes soll ein umfangreiches Beratungsangebot garantiert werden.
 	\item Der FSR sorgt durch Öffentlichkeitsarbeit für Transparenz und Anerkennung unter der Studierendenschaft. Dies wird unter anderem durch regelmäßige Information auf der Internetseite sowie durch Aushänge und Informationsveranstaltungen gewährleistet.
+	\item Protokolle über öffentliche Sitzungen des FSR werden zur Steigerung der Transparenz allen Physikstudierenden digital zugänglich gemacht
```

Comment:
Das sollte entweder in anderer Form an die Zeile davor angehangen werden oder aber mit "Der FSR.." beginnen und dann nicht das Wort "Transparenz" enthalten.